Gems/ROS2/Code/Source/Lidar/ROS2LidarSensorComponent.cpp

```diff
@@ -150,41 +158,78 @@ namespace ROS2
         PublishRaycastResults(lastScanResults.value());
     }
 
+    template<typename T>
+    using Pc2MsgIt = sensor_msgs::PointCloud2Iterator<T>;
```

Comment:
This should be moved to .h file, or to the top of this cpp file

Reply:
Or if it's used only inside one function maybe just put it inside that function

Reply:
As this is a template declaration I'm unable to put this inside the function definition. I do not think however that this is a cause for concern as this alias is local to this source file.